plugins/Carbonix/UI/EmergencyTab.cs

```diff
@@ -124,5 +124,52 @@ private void but_fenceenable_Click(object sender, EventArgs e)
             // Send aux function of 11:FENCE to high
             Host.comPort.doCommand(MAVLink.MAV_CMD.DO_AUX_FUNCTION, 11, 2, 0, 0, 0, 0, 0);
         }
+
+        private void but_arm_Click(object sender, EventArgs e)

```

Comment:
suggestion - why not have a single button where text changes between ARM/DISARM (similar to CONNECT/DISCONNECT button)

Reply:
Explicit Arm and Disarm buttons on this page are, in my opinion, safety features. It means that no matter what Mission Planner *thinks* the aircraft is doing, you always have a way to send a disarm packet, and always have a way to send an arm packet.

Reply:
Would like @LachlanConn 's opinion on this too though.

Reply:
I agree with Bob to have separate buttons especially on the emergency tab where simpler is better.